app/src/main/res/values/strings.xml

```diff
@@ -135,4 +135,18 @@
 
     <!-- TODO: Remove or change this placeholder text -->
     <string name="hello_blank_fragment">Hello blank fragment</string>
+    <string name="about_measurements">
```

Comment:
I think this bit is missing from the Hebrew version.
Also, the colored text is not working